airflow/providers/microsoft/azure/hooks/synapse.py

```diff
@@ -240,53 +296,22 @@ class AzureSynapsePipelineRunException(AirflowException):
     """An exception that indicates a pipeline run failed to complete."""
 
 
-class AzureSynapsePipelineHook(BaseHook):
+class AzureSynapsePipelineHook(BaseAzureSynapseHook):
     """
     A hook to interact with Azure Synapse Pipeline.
 
     :param azure_synapse_conn_id: The :ref:`Azure Synapse connection id<howto/connection:synapse>`.
     :param azure_synapse_workspace_dev_endpoint: The Azure Synapse Workspace development endpoint.
     """
 
-    conn_type: str = "azure_synapse_pipeline"
-    conn_name_attr: str = "azure_synapse_conn_id"
-    default_conn_name: str = "azure_synapse_connection"
-    hook_name: str = "Azure Synapse Pipeline"
-
-    @classmethod
-    def get_connection_form_widgets(cls) -> dict[str, Any]:
-        """Returns connection widgets to add to connection form."""
-        from flask_appbuilder.fieldwidgets import BS3TextFieldWidget
-        from flask_babel import lazy_gettext
-        from wtforms import StringField
-
-        return {
-            "tenantId": StringField(lazy_gettext("Tenant ID"), widget=BS3TextFieldWidget()),
-        }
-
-    @classmethod
-    def get_ui_field_behaviour(cls) -> dict[str, Any]:
-        """Returns custom field behaviour."""
-        return {
-            "hidden_fields": ["schema", "port", "extra"],
-            "relabeling": {"login": "Client ID", "password": "Secret", "host": "Synapse Workspace URL"},
-        }
+    default_conn_name: str = "azure_synapse_default"
```

Comment:
since the default connection name you are changing would it be a breaking change?

Reply:
Ah right, I understand it would be a breaking change. So, what would be the best practice to change the `default_conn_name`?

Reply:
I'll say do not remove it instead add a deprecation warning if azure_synapse_conn_id is azure_synapse_connection and then in the major version release you can remove it. ?

Reply:
okay, you mean that instead of removing the old value `default_conn_name = azure_synapse_connection`,  I should add deprecation warning and later in the new verison I must update it to new value i.e `default_conn_name = azure_synapse_default`. right?